tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -33,13 +33,43 @@ async def tx_in_pool(mempool: MempoolManager, tx_id: bytes32) -> bool:
     return True
 
 
+def assert_rpc_response(response, condition_func=None) -> Dict:
+    assert isinstance(response, dict)
```

Comment:
Generally hinting should be the first line of defense against this.  Some work is being done over in https://github.com/Chia-Network/chia-blockchain/pull/11889 to centralize the requirements and force them to be propagated, at least for fully hinted code which this apparently is not.  `:[`